content/fr/docs/intro/using_helm.md

```diff
@@ -0,0 +1,406 @@
+
+```console
+$ helm uninstall happy-panda
+```
+
+Cela supprimera la release du cluster. Vous pouvez voir toutes vos releases actuellement déployées avec la commande `helm list`:
+
+```console
+$ helm list
+NAME            VERSION UPDATED                         STATUS          CHART
+inky-cat        1       Wed Sep 28 12:59:46 2016        DEPLOYED        alpine-0.1.0
+```
+
+À partir du résultat ci-dessus, nous pouvons voir que la version `happy-panda` à bien été désinstallé.
+
+Dans les versions précédentes de Helm, lorsqu'une release était supprimée, l'historique entier restait disponible. Depuis Helm 3, la désinstallation supprime également l'historique de la release.
+Si vous souhaitez tout de même conserver un enregistrement de l'historique, utilisez `helm uninstall --keep-history`.  
+L'utilisation de `helm list --uninstalled` affichera uniquement les release qui ont été désinstallés avec l'indicateur `--keep-history`.
+
+L'indicateur `helm list --all` vous montrera tous les historique de release que Helm a conservés, y compris les historiques des éléments ayant échoué ou ayant été supprimé (si `--keep-history` était spécifié):
+
+```console
+$  helm list --all
+NAME            VERSION UPDATED                         STATUS          CHART
+happy-panda     2       Wed Sep 28 12:47:54 2016        UNINSTALLED     wordpress-10.4.5.6.0
+inky-cat        1       Wed Sep 28 12:59:46 2016        DEPLOYED        alpine-0.1.0
+kindred-angelf  2       Tue Sep 27 16:16:10 2016        UNINSTALLED     alpine-0.1.0
+```
+
+Notez que les historiques de release étant désormais supprimées par défaut, il n'est plus possible de restaurer une ressource désinstallée.
+
+## 'helm repo': Travailler avec des Dépôts
+
+Helm 3 n'est plus livré avec un dépôt de charts par défaut. La commande `helm repo` vous permet entre autre d'ajouter, de répertorier et de supprimer des dépôts.
+
+Vous pouvez voir quels dépôts sont configurés en utilisant `helm repo list`:
+
+```console
+$ helm repo list
+NAME            URL
+stable          https://charts.helm.sh/stable
+mumoshu         https://mumoshu.github.io/charts
+```
+
+Vous pouvez bien entendu ajouter de nouveaux dépôts avec `helm repo add`:
+
+```console
+$ helm repo add dev https://example.com/dev-charts
+```
+
+Étant donné que les dépôts de charts changent fréquemment, vous pouvez à tout moment vous assurer que dépot Helm est à jour en exécutant `helm repo update`.
+
+Les dépôts peuvent être supprimés avec `helm repo remove`.
+
+## Création de vos propres charts
+
+Le [Guide de développement de charts]({{< relref path="/docs/topics/charts.md" lang="en" >}}) explique comment développer vos propres charts. Mais vous pouvez vous lancer rapidement dans la création de charts avec la commande `helm create`:
+
+```console
+$ helm create deis-workflow
+Creating deis-workflow
+```
+
+Il y a maintenant un chart dans `./Deis-workflow`. Vous pouvez le modifier et créer vôtre modèles personalisé.
```

Comment:
I would recommend not capitalizing that directory, as it may confuse people (and the path will be case-sensitive).

Reply:
indeed